include/pistache/http.h

```diff
@@ -98,11 +100,32 @@ namespace Uri {
         bool has(const std::string& name) const;
         // Return empty string or "?key1=value1&key2=value2" if query exist
         std::string as_str() const;
-        
+
         void clear() {
             params.clear();
         }
 
+        // \brief Return iterator to the beginning of the parameters map
+        std::unordered_map<std::string, std::string>::const_iterator
+          parameters_begin() const {
+          return params.begin();
```

Comment:
(since this is new code), should this line be indented, to follow existing (although undocumented) style?  Same in parameters_end(), below.

Reply:
Yup, sorry. This happened as I saw last minute that you are using the style with the brackets in the same line as the function declaration. I had it with the bracket in a new line and then the indent was OK. Changing it now.